packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -61,40 +62,116 @@ export interface IBlobManagerLoadInfo {
     redirectTable?: [string, string][];
 }
 
+export type IBlobManagerRuntime =
```

Comment:
For my own education, do we typically narrow the IContainerRuntime to the subset needed by subcomponents like BlobManager, or did you have a specific reason for doing so (e.g., testing?)

Reply:
Thanks for the explaination.  I made it a comment.